client/index.jsx

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './app';
+import { Loader } from '@googlemaps/js-api-loader';
 
-ReactDOM.render(
-  <App />,
-  document.querySelector('#root')
-);
+const loader = new Loader({
+  apiKey: 'AIzaSyCG4ySJPtAS0a5rJRHZV8uPM6fgOzCZD0A',
```

Comment:
Instead of hard-coding your API key, you should put it in your `.env` file and then dynamically replace it with the Webpack Environment plugin. Instead of a string, you'd do `apiKey: process.env.GOOGLE_MAPS_API_KEY` here in your JavaScript.

Here is the Environment plugin.
https://webpack.js.org/plugins/environment-plugin/#usage

Here is how you register a plugin in your Webpack config file.
https://webpack.js.org/concepts/plugins/#usage